Source/Core/Core/Core.cpp

```diff
@@ -131,6 +132,8 @@ static Common::Event s_cpu_thread_job_finished;
 static thread_local bool tls_is_cpu_thread = false;
 static thread_local bool tls_is_gpu_thread = false;
 
+static Common::Timer time_tracker;
```

Comment:
Generally, our convention is to prefix `static` members with `s_` to make it more obvious on usage sites.

However: we also try to use as little `static` as possible, because it makes the code difficult to reuse in other places.

I wonder if we could just piggyback on the general main loop, keep track of when it was started and simply write the time when emulation ends (potentially with checkpoints every _N_ seconds/minutes so we can even track partial time) rather than using a dedicated timer just for this.

Reply:
I was unable to find the main loop which is why I don't have it update in intervals; I could avoid using a timer still by just storing the initial time emulation started and calculate the time difference in the exit method instead

Reply:
I basically mean somewhere around here (but I'm not sure if thats the best place to put it in):
https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/Core.cpp#L398-L399

And thats basically what you just suggested.

The advantage of updating it on-the-go would be surviving crashes/unclean stop, but thats something we can always look at later.

Reply:
Ah, this looks good, ty.

Reply:
I rewrote it to start a thread in the CPU Run method so the previous Core code has been removed.